docs/public/configuration/profiles.md

```diff
@@ -19,14 +22,9 @@ The content of the two files, as-is or modified, can be merged into a custom
 profile file that can be loaded by Zonemaster-Engine. Both Zonemaster-CLI and
 Zonemaster-Backend have direct options for loading a custom profile file.
 
-A custom profile file only has to contain those properties that are changed
-in comparison to the default profile file.
-
-You are adviced not to modify the default profile file in the default path on
-installed Zonemaster, since such modifications will be overwritten by updates
-of Zonemaster.
+A custom profile file only has to contain those properties that it should
+override.
```

Comment:
I think the properties should be listed, or it should link to a list of properties bacause this is quite unclear what it means.

Reply:
I added a link.